Changelog.md

```diff
@@ -12,6 +12,7 @@ Compiler Features:
 Bugfixes:
  * Antlr Grammar: Allow builtin names in ``yulPath`` to support ``.address`` in function pointers.
  * Control Flow Graph: Perform proper virtual lookup for modifiers for uninitialized variable and unreachable code analysis.
+ * IR Generator: Fix IR syntax error when copying storage arrays of structs containing functions.
  * Immutables: Fix wrong error when the constructor of a base contract uses ``return`` and the parent contract contains immutable variables.
```

Comment:
Do we sort this in case-sensitive or case-insensitive manner :-D?

Reply:
well, I used `:sort` in vim :> I can change it

Reply:
I'm not saying we should order it differently, just that I'm not sure, if we have a fixed convention for this :-).